src/pages/home/report/ReportActionsView.js

```diff
@@ -120,6 +127,13 @@ class ReportActionsView extends React.Component {
     }
 
     shouldComponentUpdate(nextProps, nextState) {
+        const policy = this.props.policies[`${ONYXKEYS.COLLECTION.POLICY}${this.props.report.policyID}`];
```

Comment:
@hungvu193  One change. This should be checked only before the icons equality check.
https://github.com/Expensify/App/blob/main/src/pages/home/report/ReportActionsView.js#L171

I don't think this is of high priority to be checked for equality first.  Can we move this check just before the icons equality check?

Reply:
I don't think there is use in fetching the policy before and checking its equality on lower part. This policy is not used anywhere else. 
can we move the policy and nextPolicy lines before the equality check?

Reply:
@hungvu193